plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1007,6 +1008,32 @@ public void testNullSensitiveTopNPushdown()
         }
     }
 
+    @Test
+    public void testArithmeticPredicatePushdown()
```

Comment:
nit: It might be cleaner to extract test to `BaseJdbcConnectorTest` as first commit and then in 2nd add arithmetic pushdown to SQL Server connector.